docs/configuration.md

```diff
@@ -313,6 +315,65 @@ Given the config above, and assuming child config “d” is selected, the follo
 3. The result of 2 will be mixed into the base config
 4. The result of 3 will be the resolved config
 
+## Configuring loaders
+
+A loader is a script used by Intern to load suites into the testing environment. Only a single loader may be specified per
+ environment (Node or browser). Intern has a number of [loaders](https://github.com/theintern/intern/tree/master/src/loaders)
+ available. If no loader is defined, then the default loaders are used per environment.
```

Comment:
Maybe make it clearer that "default loader" will mean "no loader" in the browser, which means no modules.